test/clj_sqs_extended/core_test.clj

```diff
@@ -0,0 +1,28 @@
+(ns clj-sqs-extended.core-test
+  "Basic tests for the primary API of `clj-extended-sqs`."
```

Comment:
this is an obvious pattern (i.e. `core-test` testing `core`). remove unnecessary docstring.